proxy/src/bin/nuke.rs

```diff
@@ -0,0 +1,41 @@
+use std::fs::remove_dir_all;
+use std::io::ErrorKind;
+use std::process::exit;
+
+use log::{info, trace};
+
+use proxy::coco::control;
+use proxy::config;
+use proxy::env;
+
+fn main() {
+    env::set_if_unset("RUST_BACKTRACE", "full");
+    env::set_if_unset("RUST_LOG", "info");
+    pretty_env_logger::init();
+
+    info!("Nuking application state...");
+    match remove_dir_all(config::dirs().data_dir()) {
+        Ok(_) => info!("done"),
+        Err(err) => {
+            if err.kind() == ErrorKind::NotFound {
+                info!("already gone");
+            } else {
+                trace!("{:?}", err);
+                exit(1);
+            }
+        },
+    };
+
+    info!("Nuking coco state...");
+    match control::nuke_monorepo() {
```

Comment:
I _think_ the above `remove_dir_all` would mean the monorepo is already gone no?

Reply:
No the above one nukes the directory managed by Upstream for its state.

Reply:
Ah, I suppose it's the case they're technically the same. But could be different. Great point!

Reply:
Nah they live under their own directories, so they are not the same. Upstream constructs its `ProjectDirs` from `xyz.radicle.radicle-upstream` and coco its from `xyz.radicle.radicle`. On linux that means the following two:

```
~/.local/share/radicle
~/.local/share/radicle-upstream
```

Reply:
Even better point!